.prettierrc.yml

```diff
@@ -1,3 +0,0 @@
-printWidth: 100
```

Comment:
Ember now ships with a prettier.js config so just using that instead of our yml file